package.json

```diff
@@ -1,14 +1,20 @@
 {
   "name": "@chanzuckerberg/eds",
-  "version": "13.11.0",
+  "version": "13.11.1-alpha.1",
   "description": "The React-powered design system library for Chan Zuckerberg Initiative education web applications",
   "author": "CZI <edu-frontend-infra@chanzuckerberg.com>",
   "homepage": "https://github.com/chanzuckerberg/edu-design-system",
   "license": "MIT",
   "exports": {
     ".": {
-      "import": "./lib/index.js",
-      "require": "./lib/index.cjs"
+      "import": {
+        "types": "./lib/index.d.ts",
```

Comment:
Need to define `types` here for TypeScript's `moduleResolution: "bundler"` config.